src/bp/ui-studio/src/web/components/Layout/Sidebar.jsx

```diff
@@ -46,12 +46,8 @@ class Sidebar extends React.Component {
 
   mediaQueryChanged = () => this.setState({ sidebarDocked: this.state.mql.matches })
 
-  toggleNluCollapse = event => {
-    event.preventDefault()
-    this.setState({ nluCollapseOpen: !this.state.nluCollapseOpen })
-  }
-
-  handleSideBarLeave = () => this.setState({ nluCollapseOpen: false })
+  showNluMenu = () => this.setState({ nluCollapseOpen: true })
```

Comment:
you can replace those 2 functions by a single toggle

Reply:
and well this could also have been fixed by a simple css line

Reply:
Hmm, css would have required using something else than the Collapse component.
I thought of using a single function, but it meant i had to pass the state (open or closed) as a variable, i prefer passing references in render() methods instead of an anonymous method